apps/omg_api/lib/state/transaction/fee.ex

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.API.State.Transaction.Fee do
```

Comment:
I've moved tx/fee related checks to this separate module

Reply:
There's an alternative place in the hierarchy: `OMG.API.Fees`, to avoid multiplying modules and gather fee-related stuff in a single spot